www/source/partials/docs/_reference-environment-vars.html.md.erb

```diff
@@ -18,6 +18,7 @@ This is a list of all environment variables that can be used to modify the opera
 | `HAB_ORIGIN_KEYS` | build system | no default | Comma-separated list of origin keys to automatically share with the build system |
 | `HAB_RING` | Supervisor | no default | The ring used by the Supervisor when running with [wire encryption](/docs/using-habitat#using-encryption) |
 | `HAB_RING_KEY` | Supervisor | no default | The name of the ring key when running with [wire encryption](/docs/using-habitat#using-encryption) |
+| `HAB_SECRET_<VARIABLE>` | build system | no default | Prefix to allow environment variables into the studio. The prefix will be removed and your variable will be passed into the studio at build time. |
```

Comment:
What about `HAB_STUDIO_SECRET_<VARIABLE>` to make it clear that these secrets only ever apply to the Studio (hence build time) vs. maybe applying to the Supervisor (i.e. run time).

The only thing I don't like about that above suggestion is that it's super long which can be more annoying for a user. However, given a choice, I'd rather be clearer to a user than have them type less.

In any case I wouldn't mind taking a look here before we push a set of variables up higher into a more generally useful space (that is, a variable that isn't scoped to one Habitat component more clearly).